### PR TITLE
Fixing bugs: Barreto comments

### PR DESCRIPTION
**Pull request by Melissa-Martinez-XL**
## Ticket

* Resolving Bugs

## Type of change

* [X] Fix
* [ ] Story
* [ ] Chore

## Description of the change

In the sprint review of December 19, 2022, Barreto mentioned some bugs that I have to fix, these bugs are:
* The category on the card is the ID and not the displayName as in the file that BlogList has.
* The sizes of the images are wrong, stretched.
* The whole card should be a link and not just the title.

## Screenshot/Execution
Main
image

Branch
image

## Related PRs
Links to the related PRs

### CODE REVIEW

**Comment by Melissa-Martinez-XL** [comment]
For these errors we don't have a ticket in notion, but, I wrote the description of the changes I need to make.

File: src/components/category/category.jsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ const Category = ({ data }) => (
   <div className={categoryStyles.categoryWrapper}>
     <FontAwesomeIcon className={categoryStyles.styledIcon} icon={faCircle}/>
     <p className="category">{data}</p>
+    {console.log(data)}
```
Comment (diegorosolino) [inline src/components/category/category.jsx:13]: Console.log here

File: src/components/card/card.jsx [hunk 2]
```diff
@@ -5,18 +5,28 @@ import _ from "lodash";
...
+const Card = ({ data }) => {
+
+  const category = CategoriesYAMLDATA.find(({ category }) => (category === data.frontmatter.category));
+  const categoryBlog = CategorySerializer.deSerialize(category);
+
+  return (
+    <Link className={styles.styledLink} to={`/${_.kebabCase(data.frontmatter.permalink)}`}>
+      <article key={data.id} className={styles.container}>
+        <img className={styles.styledImage} src={data.frontmatter.thumbnail}/>
+        <Category data={categoryBlog.displayName}/>
```
Comment (diegorosolino) [inline src/components/card/card.jsx:22]: Does not the data prop has this value?

Comment (Melissa-Martinez-XL) [reply src/components/card/card.jsx:22]: No, I search and see the object data, and it is not the same, for this reason I made the displayName the same as the blogList. To have the name that corresponds.

File: src/components/category/category.jsx (hunk 1 above)
Comment (Melissa-Martinez-XL) [reply src/components/category/category.jsx:13]: Done

**Comment by Melissa-Martinez-XL** [comment]
I want to know your opinion on whether the photos are now showing stretched or not.

File: src/helpers/use-category.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import CategoriesYAMLDATA from "../../content/categories.yaml";
+import { CategorySerializer } from '../serializer';
+
+const useCategory = (data) => {
+  const category = CategoriesYAMLDATA.find(({ category }) => (category === data));
+  const categoryBlog = CategorySerializer.deSerialize(category);
+  return categoryBlog;
```
Comment (diegorosolino) [inline src/helpers/use-category.js:9]:
```suggestion
  return CategorySerializer.deSerialize(category);
```

**Review by diegorosolino — APPROVED** [review]

File: src/components/card/card.module.scss [hunk 4]
```diff
@@ -7,23 +7,34 @@
   flex-wrap: wrap;
 }
 
-.styledImage {
-  border-radius: 3%;
-  margin-bottom: 1.5rem;
+.imageContainer {
+  position: relative;
+  padding-top: 56.6%;
+  display: block;
```
Comment (diegorosolino) [inline src/components/card/card.module.scss:13]: With this changes I think you don't need set width and height in this CSS class

File: src/components/card/card.module.scss [hunk 5]
```diff
@@ -7,23 +7,34 @@
...
+  left: 0;
+  height: 100%;
+  width: 100%;
+  border-radius: 3%;
+  object-fit: fill;
 
   @media screen and (max-width:50rem) {
-    height: 8.125rem;
+    height: 14rem;
   }
```
Comment (diegorosolino) [inline src/components/card/card.module.scss:30]: Does it work in mobile?

Comment (Melissa-Martinez-XL) [reply src/components/card/card.module.scss:30]: Yes

Comment (diegorosolino) [reply src/components/card/card.module.scss]: I was referring to this prop breaking the 16:9 ratio, is that ok?

Comment (Melissa-Martinez-XL) [reply src/components/card/card.module.scss]: If I delete the media, nothing happens and the value 16:9 ratio is okay on mobile.

**Review by diegorosolino — APPROVED** [review]

**Pull request merged**